src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NFloat.cs

```diff
@@ -120,13 +119,23 @@ public double Value
         /// <param name="value">The value to increment.</param>
         /// <returns>The result of incrementing <paramref name="value" />.</returns>
         [NonVersionable]
-        public static NFloat operator ++(NFloat value) => new NFloat(value._value + 1);
+        public static NFloat operator ++(NFloat value)
+        {
+            NativeType tmp = value._value;
+            ++tmp;
+            return new NFloat(tmp);
```

Comment:
Did this change because `+ 1` and `++` have different semantics?  Or just out of style consistency of ensuring the implementation of the operator uses the same operator?

Reply:
Consistency with how the other types which needed to do this where doing this.